### ISSUE TITLE
RP SHOP - 70% OFF RETAIL PRICE

### ISSUE THREAD

**Issue by Kryptonitelol** [R3nzTheCodeGOD/R3nzSkin#1129]
Hello guys, i noticed a lot of my customer came from this R3nz, so i just want to sponsor my shop, since i offer a lot of skin/pass etc really cheap price, join here: https://discord.gg/Jrp5ent8vE

**Comment by Alexrbpinto** [comment]
Yes, good shop, fast delivery, very cheap prices and nice service 
⭐⭐⭐⭐⭐

**Comment by illitiratehobo** [comment]
yeah and u two are the same person ong or friends

**Comment by Kryptonitelol** [comment]
> yeah and u two are the same person ong or friends

No... He just came from github and told me i can sponsor here. I am famous on another black market 
![image](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/169613491/18256680-92bd-41ce-90d2-89045c68ca2e)

**Comment by ceryyyy** [comment]
The link is invalid

**Comment by J-EnJay** [comment]
it should be closed @hotline1337

**Comment by Kryptonitelol** [comment]
> The link is invalid

Try this https://discord.gg/leaguechoose
